wrap/map.xml

```diff
@@ -5,9 +5,10 @@
 <authors>
     <author uuid="dff980ed-9f69-491e-b79c-30c21350baf2"/> <!--  DoomyBoy  -->
 </authors>
+<game>Capture the Wool</game>
```

Comment:
I belive this isn't necessary, and will force a non-translatable message into the scoreboard when pgm in the future has per-player scoreboards

Reply:
Done!